docs/forms.md

```diff
@@ -64,7 +64,43 @@ Form component will validate and map all this json data as if they were sent in
 
 ## Loader collections bridge
 
-Work in progress !
+With Majora Framework, we got services which can load our entities in different ways, even in form types.
+Like Doctrine entity form type, this custom entity type aim to extend `choiceType` to work with Majora loaders.
+
+First of all, create you loader and add an __alias__ on it
+```xml
+<service id="demo.entity_collection_type.loader.user" class="Demo\EntityCollectionTypeBundle\Loader\UserLoader">
+    <argument type="service" id="demo.entity_collection_type.repository.user" />
+    <tag name="majora.loader"
+        alias="demo.user.loader"
+        entity="Demo\EntityCollectionTypeBundle\Model\User"
+        collection="Demo\EntityCollectionTypeBundle\Collection\UserCollection"
+    />
+</service>
+```
```

Comment:
Some doc is missing here, we need to explain `LoaderBridgeInterface` implementation is requested into loader class to be registered.
We should say that method `choiceList()` is the default method used by form type to guess data into registered loaders.